src/SubscriptionBuilder.php

```diff
@@ -118,7 +118,7 @@ public function trialDays($trialDays)
      * @param  \Carbon\Carbon  $trialUntil
```

Comment:
We'll also need to change the method signature here and on the property above in the class like so:

```suggestion
     * @param  \Carbon\Carbon|\Carbon\CarbonImmutable  $trialUntil
```

Reply:
Done! I used `CarbonInterface` instead as it's more future-proof.